plugins/colorschemes/rose-pine.nix

```
@@ -0,0 +1,57 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+} @ args:
+with lib; let
+  cfg = config.colorschemes.rose-pine;
+  helpers = import ../helpers.nix args;
+in {
+  options = {
+    colorschemes.rose-pine = {
+      enable = mkEnableOption "rosepine";
+      package = helpers.mkPackageOption "rosepine" pkgs.vimPlugins.rose-pine;
+      style = helpers.defaultNullOpts.mkEnumFirstDefault ["main" "moon" "dawn"] "Theme style";
```

```suggestion
      enable = mkEnableOption "rosepine";

      package = helpers.mkPackageOption "rosepine" pkgs.vimPlugins.rose-pine;

      style = helpers.defaultNullOpts.mkEnumFirstDefault ["main" "moon" "dawn"] "Theme style";
```
Could you please space all options ?